src/amp-story-player/amp-story-player-impl.js

```diff
@@ -1146,6 +1150,34 @@ export class AmpStoryPlayer {
     });
   }
 
+  /**
+   * Store aspect ratio of the loaded story.
+   * @param {!StoryDef} story
+   * @private
+   */
+  storeAspectRatio_(story) {
+    if (story.iframe.contentWindow.document.documentElement) {
+      story.desktopAspectRatio = computedStyle(
+        story.iframe.contentWindow,
+        story.iframe.contentWindow.document.documentElement
+      ).getPropertyValue('--i-amphtml-story-desktop-one-panel-ratio');
```

Comment:
could we store `--i-amphtml-story-desktop-one-panel-ratio` in a const somewhere up to. similar to `CLASS_NO_NAVIGATION_TRANSITION`

Reply:
I guess we can but we actually use it once in this file though?